docs/root/configuration/http/http_filters/aws_lambda_filter.rst

```diff
@@ -4,7 +4,7 @@
 AWS Lambda
 ==========
 
-* :ref:`v2 API reference <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.aws_lambda.v3.Config>`
```

Comment:
Do we want to retain dual v2/v3 references for filters? My inclination is "no", since v2 is deprecated, but just throwing this out there to @envoyproxy/api-shepherds

Reply:
Was my reasoning too. There are a few spots in the docs where v2 and v3 are explicitly referenced -- I kept those unchanged.

Reply:
Yeah I think it's fine to bulk upgrade them.